app/src/main/kotlin/io/github/plastix/kotlinboilerplate/ui/base/PresenterActivity.kt

```diff
@@ -27,11 +27,6 @@ abstract class PresenterActivity<V : MvpView, T : Presenter<V>> : BaseActivity()
     }
 
     @CallSuper
-    protected fun onPresenterDestroyed() {
```

Comment:
This is a hook for subclasses to override if they want to perform an action when the presenter gets destroyed.